gaia-switch.js

```diff
@@ -44,11 +44,31 @@ module.exports = component.register('gaia-switch', {
     this.disabled = this.getAttribute('disabled');
     this.checked = this.getAttribute('checked');
 
+    // put non-critical operations in post-load time frame
+    // to save loadtime.
```

Comment:
Does this make sense?

 'Setting element attributes during document load can be expensive. Deferring this using setTimeout seems to improve this.'

Please add more detail if need be and update the comment :)